### ISSUE TITLE
Review guidance on success criterion 1.3.2 

### ISSUE THREAD

**Issue by mattgarrish** [w3c/epub-specs#1695]
Going back over the WCAG documents, I believe I understand the hesitance they had last time we discussed our focus on spine sequence.

Looking at [our technique](https://w3c.github.io/epub-specs/epub33/a11y-tech/#access-001), the first part seems fine. The order of the markup within a page needs to make sense ___and___ it also needs to make sense across pages because we have to deal with two-page spreads in fixed layouts.

But that's different from ordering documents in the spine and marking non-linear items. This part of the guidance really isn't related to the sequence of markup within/across pages.

If documents aren't sequenced properly, the publication is broken. Period. It's a general usability problem for everyone, not specifically an accessibility issue.

I don't see that we need to spell this out for people, so I'd suggest we remove the text and examples from the technique.

**Comment by iherman** [comment]
The issue was discussed in a [meeting](https://www.w3.org/publishing/groups/epub-wg/Meetings/Minutes/2021-06-10-epub-a11y#section3) on 2021-06-10

List of resolutions:

- ***[Resolution No. 1](https://www.w3.org/publishing/groups/epub-wg/Meetings/Minutes/2021-06-10-epub-a11y#resolution1):  approve request in issue #1695***

<details><summary><i>View the transcript</i></summary>

### 3. Review guidance on success criterion 1.3.2 

_See github issue [#1695](https://github.com/w3c/epub-specs/issues/1695)._




> *Avneesh Singh:* Avneesh_: actual text is at:cvhttps://w3c.github.io/epub-specs/epub33/a11y-tech/#access-001

**Matt Garrish:** this is about meaningful sequence of content, and meaningful sequence of documents  
… sequence of documents isn't specifically an a11y issue, it's a basic requirement for the epub to function  
… going into spine ordering and all this other stuff in this part of the Techniques might be irrelevant here  
… maybe we can just strip this stuff  

**Gregorio Pellegrino:** does it also apply to the hierarchical order of the headings?  

**Matt Garrish:** I don't think that's an issue for this section, that would fall under headings  
… this is more about having the reading order matching the visual sequence of the epub  

**Avneesh Singh:** so we keep statement that the logical reading order applies across the epub, but leave out the stuff about spine, etc.  

**Charles LaPierre:** how would this affect choose your own adventure style books where you might not have a logical progression?  

**Matt Garrish:** I think that's again a separate issue  
… in that example, there's still a reading order, its just that you decide that order based on your ability to access the links  

**Tzviya Siegman:** this just explicitly spells out what most publishers do anyway, so there should be no problem with removing the example about the spine  

> **Proposed resolution: approve request in issue #1695** *(Avneesh Singh)*

> *Tzviya Siegman:* +1

> *Gregorio Pellegrino:* +1

> *Matthew Chan:* +1

> *Bill Kasdorf:* +1

> *Will Awad:* +1

> *Cristina Mussinelli:* +1

> *Wendy Reid:* +!

> *Wendy Reid:* +1

> ***Resolution #1: approve request in issue #1695***

</details>
